src/components/common/Modal/ModalContext.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const ModalContext = React.createContext();
+
+export const ModalProvider = ModalContext.Provider;
+export const ModalConsumer = ModalContext.Consumer;
```

Comment:
```suggestion
export const {
  Provider: ModalProvider,
  Consumer: ModalConsumer
 } = React.createContext();
```

Reply:
I have a problem with that because at Modal component I need the `ModalContext` value. How can I resolve it?